jax/_src/numpy/polynomial.py

```diff
@@ -231,9 +231,66 @@ def polyval(p: Array, x: Array, *, unroll: int = 16) -> Array:
   y, _ = lax.scan(lambda y, p: (y * x + p, None), y, p, unroll=unroll)
   return y
 
-@implements(np.polyadd)
+
 @jit
 def polyadd(a1: Array, a2: Array) -> Array:
+  r"""Returns the sum of the two polynomials.
+
+  JAX implementation of :func:`numpy.polyadd`.
+
+  Args:
+    a1: Array of polynomial coefficients.
+    a2: Array of polynomial coefficients.
+
+  Returns:
+    An array containing the coefficients of the sum of input polynomials.
+
+  Note:
+
+    :func:`jax.numpy.polyadd` only accepts arrays as input unlike
+    :func:`numpy.polyadd` which accepts scalar inputs as well.
+
+  See also:
+    - :func:`jax.numpy.polysub`: Computes the difference of two polynomials.
+    - :func:`jax.numpy.polymul`: Computes the product of two polynomials.
+    - :func:`jax.numpy.polydiv`: Computes the quotient and remainder of polynomial
+      division.
+
+  Example:
+
+    ``jnp.polyadd`` does not accept scalar inputs:
+
+    >>> np.polyadd(2, 3)
```

Comment:
We don't need to show `np.polyadd` in the JAX examples. Examples should be about the JAX function, not about nitpicky details on differences between the numpy and JAX implementations.

Same for all docstrings below.

Reply:
Thanks! Removed code displaying numpy functions.